src/engine/controls/loopingcontrol.cpp

```diff
@@ -387,13 +393,18 @@ double LoopingControl::nextTrigger(bool reverse,
         if (loopSamples.start != m_oldLoopSamples.start ||
                 loopSamples.end != m_oldLoopSamples.end) {
             // bool seek is only valid after the loop has changed
-            if (loopSamples.seek) {
+            bool movedOut;
```

Comment:
This variable needs to be moved into a local scope, i.e. into the branch with the corresponding case label. Enclose the case label code into a block with curly braces to resolve build errors.

https://stackoverflow.com/questions/92396/why-cant-variables-be-declared-in-a-switch-statement